lib/tasks/permissions.rake

```diff
@@ -86,4 +85,48 @@ namespace :permissions do
       puts "#{counter} comment permissions successfully updated"
     end
   end
+
+  desc 'update users with missing permissions for like cards'
```

Comment:
Would you mind adding tests for this?
https://github.com/cybergizer-hq/retrospective/tree/master/spec/tasks